invenio/templates/deployments/worker.yaml

```diff
@@ -217,7 +223,8 @@ spec:
         {{- end }}
       volumes:
       - name: var-run-celery
-        emptyDir: {}
+        emptyDir:
+          medium: Memory
```

Comment:
Is this needed in the end? I seem to recall you said it did not fix the problem?

Reply:
Yes, you are right, but that's just half the truth. In the end with this definition we have the additional possibility of just deleting the pod. My expectation was that would work for automatic restart by Kubernetes, too. But perhaps there is a better solution we don't know yet. I would like to have that in if it does no harm because it helps operating the system.

Reply:
Sounds fair, thanks!